control-plane/plugin/src/operations.rs

```diff
@@ -42,3 +42,11 @@ pub trait ReplicaTopology {
     type ID;
     async fn topology(id: &Self::ID, output: &utils::OutputFormat);
 }
+
+/// GetBlockDevices trait.
+/// To be implemented by resources which support the 'get block-devices' operation
+#[async_trait(?Send)]
+pub trait GetBlockDevices {
```

Comment:
Do we need a command to get block devices for all nodes? Which would simply get all nodes, loop and for each issue get_blockdevices rest call?

Reply:
Should I add cluster wide as well? @tiagolobocastro @paulyoong

Reply:
Resolving for now.